src/cascade/executor/dismodel_main.py

```diff
@@ -0,0 +1,103 @@
+"""
+Entry point for running a the work of a single location in an EpiViz-AT cascade.
+"""
+import os
+from bdb import BdbQuit
+from datetime import timedelta
+from pathlib import Path
+from pprint import pformat
+from timeit import default_timer
+
+from pkg_resources import get_distribution, DistributionNotFound
+
+from cascade.core import getLoggers
+from cascade.executor.argument_parser import DMArgumentParser
+from cascade.executor.cascade_plan import CascadePlan
+from cascade.input_data.configuration import SettingsError
+from cascade.input_data.db.configuration import load_settings
+from cascade.input_data.db.locations import location_hierarchy
+from cascade.testing_utilities import make_execution_context
+from cascade.executor.estimate_location import estimate_location
+
+CODELOG, MATHLOG = getLoggers(__name__)
+
+
+def main(args):
+    start_time = default_timer()
+    execution_context = make_execution_context(gbd_round_id=5)
+    settings = load_settings(execution_context, args.meid, args.mvid, args.settings_file)
+    locations = location_hierarchy(execution_context)
+    plan = CascadePlan.from_epiviz_configuration(locations, settings)
+
+    for cascade_task_identifier in plan.cascade_jobs:
+        cascade_job, this_location_work = plan.cascade_job(cascade_task_identifier)
```

Comment:
Is there a reason not to have `plan.cascade_jobs` produce these items directly, maybe with a task_identifier also, rather than requiring the second lookup?

Reply:
Yes. Not only can the "this location work" be of considerable size, including locations for instance, but also having an identifier can make it possible to store in a file. I make an HDF or a zip file of dbs, and I identify each with its identifier.

Reply:
Cool